### PR TITLE
Add the plain text to the webapp

### PR DESCRIPTION
**Pull request by kwalcock**

### CODE REVIEW

**Comment by kwalcock** [comment]
Here's a start.  The output needs to be documented.  Is there any OpenAPI documentation for the json output of from processors (i.e., document and mentions) from any previous incantation of this project?

File: skema/text_reading/text_reading/src/main/scala/org/ml4ai/skema/text_reading/TextReadingPipelineWithContext.scala [hunk 1]
```diff
@@ -7,28 +7,33 @@ import org.ml4ai.skema.text_reading.serializer.SkemaJSONSerializer
...
 
   /**
     * Extracts mentions and runs the context engine over the extractions
     * @param inputText text to annotate
+    * @param contextWindowSize Size of the context window
     * @return extractions with scenario context
     */
-  def extractMentionsWithContext(inputText:String):Seq[Mention] ={
-    val mentions = this.extractMentions(inputText, None)._2
+  def extractMentionsWithContext(inputText: String, contextWindowSize: Int): Seq[Mention] = {
```
Comment (enoriega) [inline skema/text_reading/text_reading/src/main/scala/org/ml4ai/skema/text_reading/TextReadingPipelineWithContext.scala:18]: Ah, this is a good idea!

**Review by enoriega — APPROVED** [review]
LGTM

File: skema/text_reading/text_reading/src/main/scala/org/ml4ai/skema/text_reading/TextReadingPipelineWithContext.scala (hunk 1 above)
Comment (enoriega) [reply skema/text_reading/text_reading/src/main/scala/org/ml4ai/skema/text_reading/TextReadingPipelineWithContext.scala:18]: Not at all, this was a better solution

**Pull request merged**